src/StackAdmin/Resources/Commands.Resources/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Azure.Management.Authorization" version="2.8.0-preview" targetFramework="net452" />
```

Comment:
This should use the common authnotization package (Microsoft.Azure.PowerShell.Authorization) instead.